target/linux/ar71xx/files/arch/mips/ath79/Kconfig.openwrt

```diff
@@ -482,12 +482,24 @@ config ATH79_MACH_WPJ563
 
 config ATH79_MACH_XD3200
 	bool "YunCore XD3200 support"
-	select SOC_QCA956X
+        select SOC_QCA956X
+        select ATH79_DEV_ETH
+        select ATH79_DEV_GPIO_BUTTONS
+        select ATH79_DEV_LEDS_GPIO
+        select ATH79_DEV_M25P80
+        select ATH79_DEV_USB
+        select ATH79_DEV_WMAC
+
+config ATH79_MACH_XD6700
+	bool "YunCore XD6700 support"
+        select SOC_QCA956X
```

Comment:
It looks like the indention is wrong here and also in the parts on top of this

Reply:
Hi @hauke ,
  I believe I have corrected all the indentation errors etc.  Let me know if I have not as I am unfamiliar with git, submitting to projects.

Thanks,
        Ruairi